### ISSUE TITLE
Clarification Needed for FSDPSFTTrainer: Why is Learning Rate Multiplied by 1e3 in training_step Return Value?

### ISSUE THREAD

**Issue by NicholasPDean** [verl-project/verl#4314]
In the `FSDPSFTTrainer` class (verl/trainer/fsdp_sft_trainer.py), the `training_step(self, batch: TensorDict) method` returns the following dictionary:
```
return {
    "train/loss": step_loss.detach().item(),
    "train/lr(1e-3)": lr * 1e3,
    "train/time(s)": spend_time_per_step,
}
```
**Question**: Why is the learning rate multiplied by 1e3 before being returned as `"train/lr(1e-3)": lr * 1e3` instead of simply returning the actual learning rate value `lr`?

This arbitrary scaling factor seems unnecessary and could be confusing for users monitoring training metrics. Is there a specific reason for this implementation choice, or should the learning rate be returned as-is?

**Code Reference**: https://github.com/volcengine/verl/blob/main/verl/trainer/fsdp_sft_trainer.py#L522

**Comment by KoeYe** [comment]
The `lr` influences the model backward is controlled by `self.lr_scheduler.step()`. Here `"train/lr(1e-3)": lr * 1e3`, just make the value readable on **wandb**, no influence on the `lr` used in model backward process.

**Comment by NicholasPDean** [comment]
Thank you @KoeYe ! To confirm, the `lr * 1e3` is purely for visualization, and if we use a different logging system, say **mlflow** instead of **wandb**, that can display very small values, then there should be no reason to multiply the `lr` by `1e3`, right?

**Comment by Kirrito-k423** [comment]
This issue is being closed by the scheduled stale-issue cleanup because the latest public reply or reference from an account other than the scheduled runner (Kirrito-k423) was at 2025-12-01T18:21:38Z (mentioned by KoeYe), which is more than 6 months before the 2026-07-24 run. Evidence checked: issue page https://github.com/verl-project/verl/issues/4314, comments API https://api.github.com/repos/verl-project/verl/issues/4314/comments, and timeline API https://api.github.com/repos/verl-project/verl/issues/4314/timeline. Source for the latest non-runner interaction: https://api.github.com/repos/verl-project/verl/issues/events/21283997306. If this is still relevant, please comment with a current reproduction, evidence, or requirement context so maintainers can reopen it, or open a new issue with updated details.

本 issue 按定时清理规则关闭：最近一次来自当前定时任务账号（Kirrito-k423）之外账号的公开回复或公开引用时间为 2025-12-01T18:21:38Z（KoeYe 的 mentioned 事件），距离 2026-07-24 本轮运行已超过 6 个月。已检查证据：issue 页面 https://github.com/verl-project/verl/issues/4314、comments API https://api.github.com/repos/verl-project/verl/issues/4314/comments、timeline API https://api.github.com/repos/verl-project/verl/issues/4314/timeline。最近一次非任务账号公开互动来源：https://api.github.com/repos/verl-project/verl/issues/events/21283997306。如果该问题仍然有效，请带上当前版本的复现、证据或需求背景重新评论，维护者可以据此重开；也可以创建包含最新信息的新 issue。

This response was generated by a scheduled Codex GPT5.5 issue processing task.
